src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java

```diff
@@ -272,4 +272,13 @@ public class Dungeons {
 	@ConfigAccordionId(id = 2)
 	public String dungBatColour = "0:255:12:255:0";
 
+	@Expose
+	@ConfigOption(
+		name = "Add Pv Command Click on Party Finder Join",
```

Comment:
Need to shorten this name

Reply:
![image](https://user-images.githubusercontent.com/24389977/196657535-5cc9a346-0fd1-43c3-8bce-beac1d71f5b2.png)